docs/Quick-Start.md

```diff
@@ -79,6 +80,244 @@ See more details [here](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/
+SyslogIdentifier=victoriametrics
+Restart=always
+
+PrivateTmp=yes
+ProtectHome=yes
+NoNewPrivileges=yes
+
+ProtectSystem=full
+
+[Install]
+WantedBy=multi-user.target
+END
+```
+
+If you want to deploy VictoriaMetrics single as a Windows Service review the [running as a windows service docs]({{< ref "/single-server-victoriametrics/#running-as-windows-service" >}})
+
+6. Adjust the command line flags in the `ExecStart` line to fit your needs.
+
+The list of command line flags for VMSingle can be found [here]({{< ref "single-server-victoriametrics/#list-of-command-line-flags" >}})
+
+7. Start and enable the service by running
+
+`sudo systemctl daemon-reload && sudo systemctl enable --now victoriametrics.service`
+
+8. Check the that service started successfully 
+
+`sudo systemctl status victoriametrics.service`
+
+9. After VictoriaMetrics is Running verify VMUI is working by going to `http://<ip_or_hostname>:8428/vmui`
+
+
+### Starting VictoriaMetrics Cluster from Binaries
+
+On all nodes you will need to do the following
+
+1. Download the archive that matches your operating system and processor architecture from [github releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)
+
+For Open source it will look like
+
+  `victoria-metrics-<os>-<architecture>-v<version>-cluster.tar.gz`
+
+For Enterprise versions of VictoriaMetrics the ar will look like 
+
+  `victoria-metrics-<os>-<architecture>-v<version>-enterprise-cluster.tar.gz`
+
+In order for VictoriaMetrics Enterprise to start the, the -license flag must be set equal to a valid VictoriaMetrics key or the -licenseFile flag needs to point to a file containing your VictoriaMetrics license.
+
+2. Extract the archive to /usr/local/bin by running
+
+`sudo tar -xvf <victoriametrics-archive> -C /usr/local/bin`
+
+Replace victoriametrics-archive with the path to the archive you downloaded in step 1
+
+3. Create a user account for VictoriaMetrics
+
+`sudo useradd -s /usr/sbin/nologin victoriametrics`
+
+##### VMStorage
+
+1. Create a folder for storing VictoriaMetrics data
+
+`mkdir -p /var/lib/vmstorage && chown -R victoriametrics:victoriametrics /var/lib/vmstorage`
+
+2. Create a Linux Service for VMStorage service by running 
+
+```bash
+cat <<END >/etc/systemd/system/vmstorage.service
+[Unit]
+Description=VictoriaMetrics vmstorage service
+After=network.target
+
+[Service]
+Type=simple
+User=victoriametrics
+Group=victoriametrics
+Restart=always
+ExecStart=/usr/local/bin/vmstorage-prod -retentionPeriod=90d -storageDataPath=/var/lib/vmstorage
+
+PrivateTmp=yes
+NoNewPrivileges=yes
+ProtectSystem=full
+
+[Install]
+WantedBy=multi-user.target
+END
+```
+
+3. Adjust the command line flags in the ExecStart line to fit your needs.
+
+The list of command line flags for VMStorage can be found [here]({{< ref "cluster-victoriametrics/#list-of-command-line-flags-for-vmstorage" >}})
```

Comment:
Usually, we're using full URLs for the links because when viewing docs directly in the repository links will still be clickable and will point to relevant parts of the documentation.
Could you update `< ref >` links to full URLs?

Reply:
removed relative links